core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/machineLearning/sklearnAdvanced/KNNTrainer/SklearnAdvancedKNNParameters.java

```diff
@@ -0,0 +1,31 @@
+
+package edu.uci.ics.texera.workflow.operators.machineLearning.sklearnAdvanced.KNNTrainer;
+
+import edu.uci.ics.texera.workflow.operators.machineLearning.sklearnAdvanced.base.ParamClass;
+
+public enum SklearnAdvancedKNNParameters implements ParamClass {
+    n_neighbors("n_neighbors", "int"),
+    p("p", "int"),
+    weights("weights", "str"),
```

Comment:
why weights are treated as str?

Reply:
That is string attribute in Sklearn

![WX20240612-155940@2x](https://github.com/Texera/texera/assets/60329261/49705e44-74b8-48cf-90c1-0777477128d7)